src/main/java/com/bravo/user/validator/PaymentValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.bravo.user.validator;
+
+import com.bravo.user.exception.BadRequestException;
+import com.bravo.user.utility.ValidatorUtil;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+
+@Component
+public class PaymentValidator extends CrudValidator {
+  public static final String USER_ID_MISSING_ERROR = "'User Id' in UUID format is required";
+
+  public void validateId(String id){
+    if(!ValidatorUtil.isUUIDValid(id)){
+      throw new BadRequestException(USER_ID_MISSING_ERROR);
+    }
```

Comment:
The UUID object itself is immutable.  String is as well.  On the fence with this should return the UUID so it's potentially easier to code.  e.g. passing UUID, String into as method arguments vs String, String.  String, String may accidentally be flipped around.  Would like to know your thoughts on this one.

Reply:
Lost the context a little with this question (not sure what method has two strings in signature currently?)  - but yeah, it could be cleaner to pass in UUID to validate method than a string.